Adds standardized variables, fdtfile, fdt_addr_r, ramdisk_addr_r and
kernel_addr_r.

Index: u-boot/include/configs/udoo.h
===================================================================
--- u-boot.orig/include/configs/udoo.h
+++ u-boot/include/configs/udoo.h
@@ -110,8 +110,12 @@
 	"fdt_high=0xffffffff\0" \
 	"initrd_high=0xffffffff\0" \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
+	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"fdt_addr=0x18000000\0" \
-	"boot_fdt=try\0" \
+	"fdt_addr_r=0x18000000\0" \
+	"ramdisk_addr_r=0x13000000\0" \
+	"kernel_addr_r=" __stringify(CONFIG_LOADADDR) "\0"	\
+	"boot_fdt=try\0"	\
 	"ip_dyn=yes\0" \
 	"mmcdev=0\0" \
 	"mmcpart=1\0" \
